cmd/statusd/main.go

```diff
@@ -81,18 +80,14 @@ func main() {
 	}
 
 	backend := api.NewStatusBackend()
-	started, err := backend.StartNode(config)
+	err = backend.StartNode(config)
```

Comment:
hmm, we have this pattern in some places
```go
if err := nodeManager.StopNode(); err != nil {
	log.Printf("Failed to stop node: %v", err.Error())
  	os.Exit(1)
}
```
how do you think, maybe we can use it here too?

Reply:
but log.Fatalf exits internally (not sure why return is added)

Reply:
yeah, but I'd choose one of these 2 ways and make `main.go` consistent.
any of these is fine for me if applied consistently